### PR TITLE
#155701739 User registration and login feature

### PR DESCRIPTION
**Pull request by tonykimathi**
#### What does this PR do?
Implement TDD in the creation of user registration feature.
#### Description of Task to be completed?
Create different test cases for this feature.
Create user class and methods.
#### How should this be manually tested?
Clone the repo.
pip install nosetest
run ''nosetests tests/test_WeConnect.py -v''
#### What are the relevant pivotal tracker stories?
#155701739
#### Questions:
Feedback?

### CODE REVIEW

**Review by champagnepappi — COMMENTED** [review]
great workflow and modelling of classes

**Review by dmkitui — CHANGES_REQUESTED** [review]
Your PR does not include a command that runs tests. Kindly include an elaborate instructions in future.

File: Tests/test_WeConnect.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class UserTestCase(unittest.TestCase):
+    def setUp(self):
+        self.user = User()
+
+    def tearDown(self):
+        del self.user
+
+    def test_user_created_successfully(self):
+        response = self.user.create_user("Tom80", "tom80@email", "password80", "password80")
+        self.assertTrue(response, 'User created successfully.')
```
Comment (dmkitui) [inline Tests/test_WeConnect.py:14]: Check for the correct syntax.

**Comment by tonykimathi** [comment]
@dmkitui I have edited the PR to include the test instructions.
@champagnepappi Thanks for the feedback.

File: Tests/test_WeConnect.py (hunk 1 above)
Comment (tonykimathi) [reply Tests/test_WeConnect.py:14]: On it.

Comment (tonykimathi) [reply Tests/test_WeConnect.py:14]: @dmkitui Corrected the syntax.

**Pull request merged**